RELEASE.md

```diff
@@ -0,0 +1,50 @@
+
+
+* Create a issue to track the progress of the docker release images using the "release template" issue.
+
+* Check if the binaries of all arches are available
+  * HotSpot (x86_64, ppc64le, s390x, arm32, aarch64)
+    - It is common for hotspot arm32 and aarch64 builds to be delayed. 
+	- The usual criteria is if the arm related builds are going to be available within a couple of days of the other builds, then we wait for the builds to become available before raising the PR at the official repo. If it is expected to take longer than 2 days, we push the remaining arches right away, alerting the official maintainers of possible breakage downstream.
+  * OpenJ9  (x86_64, ppc64le, s390x)
+
+Non-Official Images
+
+
+Official Images
+
+Official images are maintained by the Docker community and updates are done through the official github repo at .... This requires an update to the Dockerfiles in this repo and a subsequent PR at the official repo with the commit id that has all the Dockerfile updates.
+
+  * Generate the updated dockerfiles
```

Comment:
How is this different to what the multiarch jobs do?

Reply:
The multiarch jobs dont update the Dockerfiles in the repo. (They update the Dockerfiles locally as part of the Jenkins job). We need to update the Dockerfiles in the repo and then point to them from the official github repo.